trading_funcs.py

```diff
@@ -91,9 +115,15 @@ def process_earnings(dates: List, diffs: List, start_date: str, end_date: str):
 
 def process_flips(ema12: np.array, ema26: np.array) -> List:
     """
-    Returns list of when 12-day ema and 26-day ema flip
-    
-    Mostly used for MACD
+    The purpose of this function is to return a list of the 12 and 26 ema flips. It
+    is primarily used for the MACD Model.
+
+    Args:
+        ema12 (np.array): The 12-day ema which is used to get the flips
+        ema26 (np.array): The 26-day ema which is used to get the flips
+
+    Returns:
+        List: The list of flips between the 12-day ema and 26-day ema. List of ints, etc.
```

Comment:
I do not think the docstring should say it a "List of ints, etc."

Once the project is typehinted, it will be obvious.
It would be better if you added something about 0 being no flip, and 1 being a flip